web/src/main/webapp/WEB-INF/data/data/resources/map/config-viewer.xml

```diff
@@ -27,5 +27,11 @@
                        opacity="1">
       <ows:Title>Bing Aerial</ows:Title>
     </ows-context:Layer>
+    <ows-context:Layer name="{type=stamen}"
```

Comment:
Maybe we could even add the layer name here with something like ```name="{type=stamen,layer=watercolor}"``` or ```name="{type:'stamen',layer:'watercolor'}"``` as this notation is an extension we made ? similar to what is done when saving a context with WMTS ... @fgravin any thoughts ?